src/gui/tiledscene/qgstiledscenesourceselect.cpp

```diff
@@ -93,6 +93,8 @@ void QgsTiledSceneSourceSelect::btnNewCesium3DTiles_clicked()
 
     QgsTiledSceneProviderConnection::addConnection( nc.connectionName(), connectionData );
     populateConnectionList();
+    QgsTiledSceneProviderConnection::setSelectedConnection( nc.connectionName() );
```

Comment:
QgsSensorThingsSourceSelect and QgsVectorTileSourceSelect likely need a similar fix

Reply:
Indeed. This is now done.